docs/DeveloperGuide.md

```diff
@@ -290,43 +290,277 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+#### Use case: Save a student's contact
```

Comment:
Looks good overall, but I think everyone should look through it to ensure its ok